pinner/pinmgr.go

```diff
@@ -356,8 +366,11 @@ func (pm *PinManager) enqueuePinOp(po *PinningOperation) {
 	if po.SkipLimiter {
 		u = 0
 	}
-
-	_, err = pm.pinQueue.EnqueueObject(getUserForQueue(u), po)
```

Comment:
@snissn can we add some form of validation on the addrs before enqueuing, so we don't fail and lose the message? Let's validate, if it fails validation, set the address to `[]`, so we still process the message, and ignore the bad addrs.

Reply:
Yes. Lets prioritize this today and get it merged asap. I’d like to do an emergency release with this fix.

Reply:
I’m OK with that. @en0ma?

Let's get this merged.

Reply:
ok